admin/src/components/RFPs/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { view } from 'react-easy-state';
+import { Button, List, Popconfirm, Spin, Tag, Tooltip, message } from 'antd';
+import { RouteComponentProps, withRouter } from 'react-router';
+import { Link } from 'react-router-dom';
+import { RFP_STATUSES, getStatusById } from 'util/statuses';
+import store from 'src/store';
+import './index.less';
+import { RFP } from 'src/types';
+
+type Props = RouteComponentProps<any>;
+
+interface State {
+  deletingId: number | null;
+}
+
+class RFPs extends React.Component<Props, State> {
+  state: State = {
+    deletingId: null,
+  };
+
+  componentDidMount() {
+    this.fetchRFPs();
+  }
+
+  render() {
+    const { rfps, rfpsFetching, rfpsFetched } = store;
+    const loading = !rfpsFetched || rfpsFetching;
+
+    return (
+      <div className="RFPs">
+        <div className="RFPs-controls">
+          <Link to="/rfps/new">
+            <Button>Create new RFP</Button>
+          </Link>
+          <Button title="refresh" icon="reload" onClick={this.fetchRFPs} />
+        </div>
+        <List
+          className="RFPs-list"
+          bordered
+          dataSource={rfps}
+          loading={loading}
+          renderItem={this.renderRFP}
+        />
+      </div>
+    );
+  }
+
+  private fetchRFPs = () => {
+    store.fetchRFPs();
+  };
+
+  private renderRFP = (rfp: RFP) => {
+    const { deletingId } = this.state;
+    const actions = [
+      <Link key="edit" to={`/rfps/${rfp.id}/edit`}>
+        <a key="edit">edit</a>
```

Comment:
While reviewing Pt. 2, I noticed a nasty-gram from the console about nested <a>, everything seems to work, but could cause event bubbling issues or sth.

Reply:
Ah you know what, this was a bad habit from the Zeit Next days. They had a link component that required you to put your own a tag in. I'll remove this.